glslc/README.asciidoc

```diff
@@ -135,9 +138,13 @@ Shader Stage Selection>> table.
   extension, the file extension is ignored and the `-fshader-stage=` argument
   is used instead.
 
-WARNING: `-fshader-stage=` overrides file extension; that means it should not
+[[shader-stage-with-spirv-assembly]]
+CAUTION: `-fshader-stage=` overrides file extension; that means it should not
 be used together with SPIR-V assembly files because glslc will treat the given
-SPIR-V assembly files as of GLSL source code of the given shader stage.
+SPIR-V assembly files as of GLSL source code of the given shader stage. If you
+need to supply both GLSL source SPIR-V assembly files as inputs to glslc, please
```

Comment:
Reword: "need to both supply SPIR-V assembly files and `-fshader-stage` on the same command line, please"